INSTALL.rst

```diff
@@ -1,6 +1,249 @@
+  python3 -m pip install -U pip
+  # Install scikit-image
+  python3 -m pip install -U scikit-image
+
+
+This is the easiest way of obtaining the latest released version of
+``scikit-image``.
+
+We also offer an easy way to install related packages. These are often included
+in scientific python distributions. To install many of them using ``pip`` use
+the command:
+
+.. code-block:: bash
+  python3 -m pip install scikit-image[optional]
+
+
+A note on virtual environments
+++++++++++++++++++++++++++++++
+
+Scientific Python Distributions listed in the first section take care to only
+install compatibile packages together. Often conflicts can arise between
+specific combinations of packages. To avoid breaking your python environments,
+we recommendg learning about virtual environments in python. You can learn more
+about them by reading Python's official documentation on `Virtual Environments
+and Packages <https://docs.python.org/3/tutorial/venv.html>`_.
+
+
+Known challenges
+++++++++++++++++
+
+1. Different python installations can cause confusion. To check which python
+   executable you are using, use the commands:
+
+
+.. code-block:: bash
+  # Linux and MAC
+  which python3
+  # Windows
+  where python3
+
+From your python console or jupyter notebook, you may use the python commands:
+
+.. code-block:: python
+
+  import sys
+  print(sys.executable)
+
+To make sure the executable used matches your expectations.
+
+
+2. Never use ``sudo`` when installing packages with ``pip``. This may cause
+   ``pip`` to overwrite important files for your operating system which may
+   require you to completely reinstall your system.
+
+
+3. ``conda`` based installation
+-------------------------------
+
+`conda <https://docs.conda.io>`_ can be a useful package manager for building
+applications with dependencies not available on pypi in a cross-platform
+setting. While detailed instructions on how to use conda are considered out of
+scope in this tutorial we point you to:
+
+- `conda's official documnetation <https://docs.conda.io>`_
+- `conda-forge <https://conda-forge.org>`_ a channel maintained with the latest scikit-image package.
+- `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_
+
+4. System wide installation
+---------------------------
+
+**Linux only:** If you must use your system's package manager to install
+scikit-image, we recommend reading the official documentation from your
+operating system. Each operating system, and Linux distribution, has its own
+package manager, and as such the exact installation steps might differ for each
+system.
+
+We warn that this may install an older version of scikit-image which we may not
+support anymore. For these reasons, we **do not** recommend using the following
+strategy to install ``scikit-image``.
+
+For brevity, we provide instructions for getting started with Ubuntu.
+To install scikit-image, you may use the command:
+
+.. code-block:: bash
+  sudo apt install python3-skimage
```

Comment:
If we include this, I'd argue that `conda install...` should also be covered.

Reply:
I'm happy to take this out, and point to ubuntu documentation too.